source/extensions/filters/http/cache/http_cache.h

```diff
@@ -213,7 +212,8 @@ class LookupRequest {
   // storage implementation forwards lookup requests to a remote cache server that supports *vary*
   // headers, that server may need to see these headers. For local implementations, it may be
   // simpler to instead call makeLookupResult with each potential response.
-  HeaderVector vary_headers_;
+  // Warning: this should not be accesses out-of-thread!
```

Comment:
@toddmgreer is this comment enough?

Reply:
It's enough, but it should be on getVaryHeaders.
s/accesses/accessed/